features/support/time_machine_helpers.rb

```diff
@@ -9,12 +9,15 @@ module TimeMachineHelpers
   # 1. Remove the user
   # 2. time_machine 1 month from now
   # 3. Assert the user has been effectively removed
-  def time_machine(till)
-    freeze_time
-
-    while Time.zone.now < till
-      travel_to(1.day.from_now)
-      run_jobs
+  def time_machine(till, opts = {})
+    opts ||= {}
+    unless opts[:skip_jobs]
```

Comment:
I commented on it [here](https://github.com/3scale/porta/pull/3515/files#r1293389796)

Basically, when we do "time travelling" all jobs are executed (all daily jobs). I noticed, that it takes too long when travelling within several years, so added this options to skip jobs, as it's not needed for this test.

In fact, I am thinking we may switch to "no jobs" by default, and only enable it (by "(with jobs)" wherever needed. I think this might reduce the time for cucumbers significantly

Reply:
I don't think we need to modify `time_machine` because if we want to travel in time without running jobs we can use `travel_to`. The point of `time_machine` is precisely to simulate the actual pass of time with jobs updating invoices, etc.